util/Migrator/DbScripts/2025-01-16_01_LimitItemDeletion.sql

```diff
@@ -0,0 +1,487 @@
+
+-- Add Columns
+IF COL_LENGTH('[dbo].[Organization]', 'LimitItemDeletion') IS NULL
+BEGIN
+    ALTER TABLE
+        [dbo].[Organization]
+    ADD
+        [LimitItemDeletion] BIT NOT NULL CONSTRAINT [DF_Organization_LimitItemDeletion] DEFAULT (0)
```

Comment:
Note: This will update all existing records in `[dbo].[Organization]`. If this table experiences heavy traffic, it could cause issues. I have leaned on DBOps here since they have more visibility into our infrastructure.